ci/tests/Jenkinsfile.e2e-nightly

```diff
@@ -48,6 +48,7 @@ pipeline {
           dir('test/appium/tests') {
             sh """
               python3 -m pytest -m testrail_id \
+                -k "not upgrade" \
```

Comment:
It should work with -k for our needs now but I'd not limit us with having word 'upgrade' in test names. Let's use in marks:
python3 -m pytest -m "testrail_id or not upgrade"

Reply:
Or even just python3 -m pytest -m "not upgrade"